geopandas/tests/test_geodataframe.py

```diff
@@ -1510,3 +1512,47 @@ def test_geodataframe_crs_colname():
     assert gdf.crs is None
     assert gdf["crs"].iloc[0] == 1
     assert getattr(gdf, "crs") is None
+
+
+@pytest.fixture
+def nybb2(nybb_filename):
+    yield read_file(nybb_filename).head(2)
```

Comment:
I know this is copied from the notebook, but would prefer for new tests to use a small dummy geodataframe (to avoid having our tests depend too much on IO). Could reuse the `dfs` fixture and just use one of both dataframes.

Reply:
Yep, totally agree - but I wanted to try and make as clear as possible what was actually changing. If we can have some consensus on what the implementation should be, then I'll simplify these tests / pinch the tests from your PR.